src/modules/feedback/components/FeedbackBox.js

```diff
@@ -36,16 +36,26 @@ type PropsType = {|
   comment: string,
   onCommentChanged: SyntheticInputEvent<HTMLTextAreaElement> => void,
   onFeedbackOptionChanged: FeedbackVariant => void,
-  onSubmit: () => void,
+  onSubmit: (value?: string) => void,
   t: TFunction,
   closeFeedbackModal: () => void,
-  theme: ThemeType
+  theme: ThemeType,
+  feedbackSent: string
 |}
 
 /**
  * Renders all necessary inputs for a Feedback and posts the data to the feedback endpoint
  */
 export class FeedbackBox extends React.PureComponent<PropsType> {
+  constructor (props: PropsType) {
+    super(props)
+    this.handleClick = this.handleClick.bind(this)
```

Comment:
`bind` is only required when using normal functions, e.g. `handleClick () { .... }`. It is not required if you are using arrow functions.

Reply:
Constructor and bind() removed